app/javascript/components/App.js

```diff
@@ -82,7 +82,7 @@ class App extends React.Component {
             let recipe = this.state.recipes.find(recipe => recipe.id === +id)
             return <RecipeShow recipe={ recipe } /> }} />
             <Route path="/userrecipes" component={ UserRecipes } />
-             <Route path="/aboutus" component={ AboutUs } />
+          <Route path="/aboutus" component={ AboutUs } />
```

Comment:
the indentation of this route should be the same as the route below

Reply:
Done